helm_deploy/values-prod.yaml

```diff
@@ -39,15 +39,6 @@ generic-service:
     FEATURE_EVENT_SENTENCE_DELETED: false
     FEATURE_EVENT_COURT_APPEARANCE_UPDATED: false
     FEATURE_EVENT_LEGACY_COURT_CASE_REFERENCES_UPDATED: false
-    FEATURE_EVENT_CONTACTS-API_CONTACT_CREATED: false
```

Comment:
ironically half the switches were missing anyway, although we are not expecting any events until May when DPS UI switched on